packages/v4/src/content/design-guidelines/components/file-upload-multiple/file-upload-multiple.md

```diff
@@ -0,0 +1,81 @@
+---
+id: File upload - multiple
+section: components
+---
+
+The **file upload - multiple** component allows users to select 1 or more files to upload to a specific location. The component can be configured to support any file type as well as restrict the user's ability to upload 1 or more files. Users have the ability to select files from their system either natively using a file input or via drag and drop. Once uploaded, the user can edit any files.
+
+A file upload component is commonly found in forms, but can also live as a standalone element.
+
+*Note: This component is for both single and multiple file upload scenarios that do not require a text preview. For a single file upload that supports text preview, see the [file upload component](/components/file-upload).*
+
+
+## Elements
+
+### Multiple file upload component
+
+<img src="./img/elements.png" alt="Multiple File Upload" width="638"/>
+
+Most multiple file upload components contain 5 basic elements:
+
+1. **Label:** Text to inform users on how a user can interact with the component. The text should describe that you can either drag and drop a file into the drop zone or click on the button to upload a file.
+
+2. **Description:** Assistive text to help the user make an informed selection on file type and size.
+
+3. **Button:** A button to upload files if preferred.
+
+4. **Accordion text:** Describes the total number of files uploaded and if they are complete or in progress.
+
+5. **Progress upload:** The name of the uploaded file, along with file size, progress bar, and percentage.
+
+## Usage/Best practices
+
+**When to use:**
+* Uploading 1 or more files.
+
+* Uploading files by dragging and dropping.
+
+* Showing the process of uploading. If large file sizes will be supported in your context, showing progress is important. However, if the file size supported will always upload in a matter of a few seconds this may not be necessary.
+
+**Best practices:**
+
+<img src="./img/best-practices.png" alt="best practices for file upload" width="1145"/>
```

Comment:
Update image to be 2x PNG instead of 1x PNG!

Reply:
I made this update. Please retest once the build completes.